geeknote/geeknote.py

```diff
@@ -791,11 +791,11 @@ def find(self, search=None, tags=None, notebooks=None,
 
         # Evernote api will only return so many notes in one go. Checks for more
         # notes to come whilst obeying count rules
-        while ((result.totalNotes != len(result.notes)) and count != 0):
+        while ((result.totalNotes != len(result.notes)) and update_count(count) != 0):
             offset = len(result.notes)
-            result.notes += self.getEvernote().findNotes(request, count,
+            result.notes += self.getEvernote().findNotes(request, update_count(count),

```

Comment:
I found that the original method can not get the original method to get all the data, each cycle when the missing data.